src/calculateRentalCost.js

```diff
@@ -4,7 +4,25 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  const totalCost = days * dailyRate;
+  const smallDiscount = 20;
+  const discount = 50;
```

Comment:
The names `smallDiscount` and `discount` are not descriptive enough to understand what they represent without reading the code. Consider renaming them to reflect the conditions under which they apply, such as `SHORT_TERM_DISCOUNT` and `LONG_TERM_DISCOUNT`. Following the checklist, avoid using literals in expressions and use constants explaining these values.